pkgs/development/ocaml-modules/janestreet/0.14.nix

```
@@ -595,7 +595,7 @@ with self;
 
   ppx_sexp_conv = janePackage {
     pname = "ppx_sexp_conv";
-    version = "0.14.3";
+    version = "0.15.0";
```

```suggestion
    version = "0.15.1";
```
https://opam.ocaml.org/packages/ppx_sexp_conv